src/main/java/session_17_equals_hashcode_generics/challenge/challenge_12/Challenge12.java

```diff
@@ -0,0 +1,36 @@
+package session_17_equals_hashcode_generics.challenge.challenge_12;
+
+public class Challenge12<K, V, N> {
+    private K key;
```

Comment:
As mentioned above, please consider leaving an empty row below the class declaration. You can consider this remark for all the classes from your repository.